### PR TITLE
Support ignore new files in subset

### PR DESCRIPTION
**Pull request by Konboi**
In Launchable, it takes time until the execution time of a newly added file is estimated
But since they are local, you may want to ignore those files when you run the test

So I added an option `--ignore-new-tests` to support this

### CODE REVIEW

File: docs/resources/cli-reference.md [hunk 1]
```diff
@@ -175,6 +175,7 @@ launchable subset [OPTIONS] TESTRUNNER ...
...
+| `--ignore-new-tests`      | Ignore that were added recently. NOTICE: this option will ignore tests that you added just now, too.                                                                                                                                                                                                                                                                                                                                                                                                                                      | No                                                        |
```
Comment (shibuiwilliam) [inline docs/resources/cli-reference.md:178]: [grammer]
`Ignore tests that were added recently. NOTICE: this option will ignore tests that you added just now as well.`

File: launchable/commands/subset.py [hunk 2]
```diff
@@ -93,6 +93,12 @@
     """,
     is_flag=True
 )
+@click.option(
+    "--ignore-new-tests",
+    "ignore_new_tests",
+    help='Ignore tests that were added recently.\n\nNOTICE: this option will ignore tests that you added just now, too',
```
Comment (shibuiwilliam) [inline launchable/commands/subset.py:99]: https://github.com/launchableinc/cli/pull/400/files#r842393927

**Review by shibuiwilliam — APPROVED** [review]
just a grammatical error

File: docs/resources/cli-reference.md (hunk 1 above)
Comment (Konboi) [reply docs/resources/cli-reference.md:178]: Ops... Thank you!

**Review by draftcode — APPROVED** [review]

**Pull request merged**